softwarelist.csv

```diff
@@ -1,104 +1,110 @@
 Software Name,B30A PCs,B30B PCs,Circ Dell Laptops,Circ Macbook Pros,Email Kiosks,iMacs,Research PCs,Stone Classroom PCs
-7-Zip, X, X, X,, X,, X, X
-Access 2016 (MS Office), X, X, X,,,, X, X
-Adobe Acrobat DC, X, X, X, X,, X, X, X
-Adobe After Effects 0217, X, X,,,,, X, X
-Adobe Animate 2017, X, X,,,,, X, X
-Adobe Audition 2017, X, X,,,,, X, X
-Adobe Bridge 2017, X, X,,,,, X, X
-Adobe Dreamweaver 2017, X, X,,,,, X, X
-Adobe Fireworks CS6, X, X,,,,, X, X
-Adobe Flash, X, X,,,,, X, X
+7-Zip, X, X, X,,,, X, X
+Access 2016 (MS Office),,, X,,,,,
+Access 2019 (MS Office), X, X,,,,, X, X
+Adobe Acrobat DC, X, X, X, X, X, X, X, X
+Adobe Acrobat Reader DC, X, X, X,,,, X, X
+Adobe After Effects CC 2018, X, X,,,, X, X, X
+Adobe Animate CC 2018, X, X,,,, X, X, X
+Adobe Audition CC 2018, X, X,,,, X, X, X
+Adobe Bridge CC 2018, X, X,,,, X, X, X
+Adobe Dreamweaver CC 2018, X, X,,,, X, X, X
+Adobe Fireworks CS6, X, X,,,, X, X, X
+Adobe Flash, X, X, X, X, X, X, X, X
 Adobe Flash Builder 4.7, X, X,,,,, X, X
-Adobe Illustrator 2017, X, X,,,, X, X, X
-Adobe InCopy 2017, X, X,,,,, X, X
-Adobe InDesign 2017, X, X,,,, X, X, X
-Adobe Lightroom , X, X,,,,, X, X
-Adobe Media Encoder 2017, X, X,,,,, X, X
-Adobe Muse 2017, X, X,,,,, X, X
-Adobe Photoshop 2017, X, X,,,, X, X, X
-Adobe Prelude 2017, X, X,,,,, X, X
-Adobe Premier Pro 2017, X, X,,,,, X, X
-Adobe Reader, ,,,, X,,,
-Adobe Speed Grade 2015, X, X,,,,, X, X
-Anaconda 3, X, X,,,,, X, X
-ArcGIS Desktop 10.5.1, X, X,,,,, X, X
-ArcGIS Pro 2.2, X, X,,,,, X, X
+Adobe Fuse CC(Beta), X, X,,,, X, X, X
+Adobe Illustrator CC 2018, X, X,,,, X, X, X
+Adobe InCopy CC 2018, X, X,,,, X, X, X
+Adobe InDesign CC 2018, X, X,,,, X, X, X
+Adobe Lightroom Classic CC, X, X,,,, X, X, X
+Adobe Media Encoder CC 2018, X, X,,,, X, X, X
+Adobe Photoshop CC 2018, X, X,,,, X, X, X
+Adobe Prelude CC 2018, X, X,,,, X, X, X
+Adobe Premier Pro CC 2018, X, X,,,, X, X, X
+Adobe Speed Grade CC 2015, X, X,,,, X, X, X
+Anaconda 2019.3, X, X,,,,, X, X
+ArcGIS 10.7.1, X, X,,,,, X, X
+ArcGIS Pro 2.4, X, X,,,,, X, X
 Artemis, X, X,,,,, X, X
-Audacity,,,, X,,,,
-Autodesk (AutoCAD) 2019, X, X,,,,, X, X
+Audacity,,,, X,, X,,
+Autodesk (AutoCAD) 2020, X, X,,,,, X, X
 CBGP - GeneClass2, X, X,,,,, X, X
-Corpscon, X, X,,,,, X, X
 Density 5, X, X,,,,, X, X
 Distance 7.1, X, X,,,,, X, X
-DVD Player,,,, X,, X,,
+DNRGPS, X, X,,,,, X, X
 EasyPop, X, X,,,,, X, X
-EndNote X7,,,,,, X,,
-EndNote X8, X, X,, X,,, X, X
+EndNote X8,,,,, X, X,,,

```

Comment:
Morning @sew268. Looks like we have an [extra column in for EndNote X8](https://github.com/cul-it/mann-softwarelist-csv/blob/dev/softwarelist.csv).

Reply:
changes made